sphinx/registry.py

```diff
@@ -132,6 +132,10 @@ def __init__(self) -> None:
         # private cache of Docutils Publishers (file type -> publisher object)
         self.publishers: dict[str, Publisher] = {}
 
+    @property
+    def autodoc_attrgettrs(self) -> dict[type, Callable[[Any, str, Any], Any]]:
+        return self.autodoc_attrgetters
```

Comment:
```suggestion
		"""Backward-compatible alias to 'autodoc_attrgetters'."""
        return self.autodoc_attrgetters
```

Do you want to add a warning?

Reply:
I'd merge a PR but I'm just trying to clear the backlog of 'easy' PRs currently.